config/olm-catalog/ibm-spectrum-scale-csi-operator/2.4.0/ibm-spectrum-scale-csi-operator.v2.4.0.clusterserviceversion.yaml

```diff
@@ -3,12 +3,12 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   annotations:
-    alm-examples: >-
+    alm-examples: |
         [{"status": {}, "kind": "CSIScaleOperator", "spec": {"provisionerNodeSelector":[{"key": "scale", "value": "true"}], "clusters":[{"secrets": "secret1", "restApi": [{"guiHost": "< Primary cluster GUI IP/Hostname>"}], "secureSslMode": false, "primary": {"primaryFs": "< Primary Filesystem >"}, "id": "< Primary Cluster ID - WARNING - THIS IS A STRING NEEDS YAML QUOTES! >"}], "pluginNodeSelector": [{"key":"scale", "value": "true"}], "resizerNodeSelector": [{"key":"scale", "value": "true"}], "attacherNodeSelector": [{"key": "scale", "value":"true"}], "snapshotterNodeSelector": [{"key": "scale", "value":"true"}]}, "apiVersion": "csi.ibm.com/v1","metadata": {"labels": {"release": "ibm-spectrum-scale-csi-operator", "app.kubernetes.io/name":"ibm-spectrum-scale-csi-operator", "app.kubernetes.io/instance": "ibm-spectrum-scale-csi-operator","app.kubernetes.io/managed-by": "ibm-spectrum-scale-csi-operator"}, "name":"ibm-spectrum-scale-csi"}}]
     capabilities: Basic Install
     categories: Storage
     certified: 'false'
-    containerImage: quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-operator:v2.4.0
+    containerImage: quay.io/ibm-spectrum-scale/ibm-spectrum-scale-csi-operator:v2.5.0
```

Comment:
We don't need to update 2.4.0 CSV

Reply:
yeah, removed it. The file got update during unit testing and I pushed the changes. Thanks.